neofetch

```diff
@@ -2772,6 +2781,14 @@ dynamicprompt() {
     # Get cursor position
     info_height="$(IFS=';' builtin read -srdR -t 1 -d c -p $'\033[6n\033[c' ROW COL; printf "%s" "${ROW#*[}")"
 
+    # On Windows 10 subsystem for Linux, the default terminal does not
+    # produce any integer output, so a workaround is used.
+    if [[ $info_height =~ ^-?[0-9]+$ ]]; then
```

Comment:
Something like this

![virtualbox_windows 10_21_09_2016_10_09_15](https://cloud.githubusercontent.com/assets/15692230/18696567/5b5501ca-7fe4-11e6-9bec-97183e0294ee.png)